packages/astro/test/astro-minification-html.test.js

```diff
@@ -0,0 +1,79 @@
+import { expect } from 'chai';
+import { loadFixture } from './test-utils.js';
+import testAdapter from './test-adapter.js';
+
+const NEW_LINES = /[\r\n]+/gm;
+
+/**
+ * The doctype declaration is on a line between the rest of the HTML.
+ * This function removes the doctype so that we can check if the rest of the HTML is without
+ * whitespace.
+ */
+function removeDoctypeLine(html) {
+	return html.slice(20);
+}
+
+/**
+ * In the dev environment, two more script tags will be injected than in the production environment
+ * so that we can check if the rest of the HTML is without whitespace
+ */
+function removeDoctypeLineInDev(html){
+	return html.slice(-100)
+}
+
+describe('HTML minification', () => {
+	describe('in DEV enviroment', () => {
+		let fixture;
+		let devServer;
+		before(async () => {
+			fixture = await loadFixture({
+				root: './fixtures/minification-html/',
+			});
+			devServer = await fixture.startDevServer();
+		});
+
+		after(async () => {
+			devServer.stop();
+		});
+
+		it('should emit compressed HTML in the emitted file', async () => {
+			let res = await fixture.fetch(`/`);
+			expect(res.status).to.equal(200);
+			const html = await res.text();
+			expect(NEW_LINES.test(removeDoctypeLineInDev())).to.equal(false);
```

Comment:
```suggestion
			expect(NEW_LINES.test(removeDoctypeLineInDev(html))).to.equal(false);
```

The `html` wasn't passed in here, causing the test to fail.